build.sbt

```diff
@@ -67,3 +67,5 @@ testOptions in Test += Tests.Argument("-oD")
 scalastyleSources in Test ++= {(unmanagedSourceDirectories in Integration).value}
 
 mainClass in Compile := Some("io.iohk.ethereum.App")
+
+coverageExcludedPackages := "io.iohk.ethereum.vmrunner.*"
```

Comment:
it is just relative change it increased by 4% comparing to `phase/3/txExecution` but decreased by 0.1 comparing to build that has aggregate coverage with it tests